js/components/conceptset/ConceptSetStore.js

```diff
@@ -130,6 +132,7 @@ define([
     }
     
     async refresh(mode) {
+	  this.currentConseptSetTab(mode);
```

Comment:
Is this necessary?  the change to currentCnceptSetTab() should result in a call to refresh(mode), but then refresh mode wants to reset currentConceptSetTab().   What issue were you trying to fix when you added this?

Reply:
I use `this.currentConseptSetTab` in DataSourceComponent for reload table data.